src/pages/algorithms/divide-and-conquer-algorithms/index.md

```diff
@@ -2,14 +2,34 @@
 title: Divide and Conquer Algorithms
 ---
 ## Divide and Conquer Algorithms
+as the name suggests, these algorithms work by breaking down a problem into smaller subproblems. This algorithmic paradigm consistis of following three general steps:-
+1. **Divide** - break the given problem
+2. **Conquer** - Solve those smaller subproblems
+3. **Combine** - Alas, put together the smaller pieces
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/algorithms/divide-and-conquer-algorithms/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### How do they work?
+before speaking the asymptotic language of Theoretical Computer Science, let's take a look at this Illustration to get a gist of how such an algorithms works!
```

Comment:
As these are intro docs, words like "asymptotic" would benefit from a quick definition, where there is no other more common word.